<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressSettings</class>
 <widget class="QWidget" name="ProgressSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QComboBox" name="cbxProgressType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Determinate</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Indeterminate</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="sldValue">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblValue">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <spacer name="vspBottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblProgressType">
     <property name="text">
      <string>Progress type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="chkDisabled">
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="MaterialProgress" name="progress">
     <property name="progressType">
      <enum>Material::DeterminateProgress</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialProgress</class>
   <extends>QProgressBar</extends>
   <header location="global">QtMaterialWidgets/progress.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
